### ISSUE TITLE
Known Bugs

### ISSUE THREAD

**Issue by boypt** [boypt/simple-torrent#4]
* files listed in `Download` may be duplicated when torrent contains multiple file during initating.
* changes to config in web UI may not successful

**Comment by snowie2000** [comment]
The file browser in the UI listed all the files on start and push the whole to the client to display which is inappropriate. 

Plus it has a limitation of 1000 files in total. If you have more than 1000 files in the download folder, some of them will be omitted and the display is corrupt.

**Comment by boypt** [comment]
> The file browser in the UI listed all the files on start and pushed the whole to the client to display which is inappropriate.
> 
> Plus it has a limitation of 1000 files in total. If you have more than 1000 files in the download folder, some of them will be omitted and the display is corrupt. If you remove this limitation, all files are displayed correctly but very slowly if the number is too big due to the first problem.

yes, this is a problem of the original design. It's better to keep the download folder clean after download.

**Comment by snowie2000** [comment]
@boypt Do you have any plan rewriting this part?

**Comment by boypt** [comment]
@snowie2000 no, not at the moment, I'm not sure what to change. The original idea is that with the help of velox library, automaticly transfer differentces of the filesystem, and display the whole directory tree in web. Too much files reusult in chaos in the UI thus adds a limitation of 1000 files max.

Removing the mechanism also removes the realtime update feature, and that needs another UI interative designing, I think that's too much work.

**Comment by Dibbyo456** [comment]
When you push new version does it auto upgrade?
